react-assissans/src/components/MovieItem.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { Card, Badge } from "react-bootstrap";
+
+function MovieItem(props) {
+  const { item } = props;
+  const img = `https://image.tmdb.org/t/p/w500${item.poster_path}`;
```

Comment:
I would prefer if it's done in this way: 
`<img src={IMG_BASE_URL + item.poster_path} alt="img"></img>`
in the card body